README.md

```diff
@@ -30,14 +30,16 @@ yarn add @ekino/logger
 
 ## Usage
 
-By default, the logger doesn't output anything. You need to define a log `level` and enabled `namespaces`
+By default, the logger output warn and error levels for all namespaces.
+You can override it to define a log `level` and log level per `namespace`
```

Comment:
You have the ability to set another global log `level` and/or add overrides per `namespace`.